packages/astro/src/@types/astro.ts

```diff
@@ -847,7 +847,7 @@ export interface AstroUserConfig {
 		 * ```js
 		 * {
 		 * 	build: {
-		 *		inlineStylesheets: `auto`,
+		 *		inlineStylesheets: `never`,
```

Comment:
Heads up that I also updated here, since we typically show a setting that would *need to be set*, and you wouldn't need to set `auto` specifically now that it's the default.

Reply:
Saving me again!
🙏